lib/rex/socket/range_walker.rb

```diff
@@ -417,6 +428,17 @@ def expand_nmap(arg)
     return ranges
   end
 
+  protected
+
+  def valid_cidr_chars(arg)
+    return false if arg.include? ',-' # Improper CIDR notation (can't mix with 1,3 or 1-3 style IP ranges)
+    return false if arg.scan("/").size > 1 # ..but there are too many slashes
+    ip_part,mask_part = arg.split("/")
+    return false if ip_part.nil? or ip_part.empty? or mask_part.nil? or mask_part.empty?
```

Comment:
`or`?

Reply:
Line was copy pasta'd, I can bring it to the future as well.